bookOfHb.md

```diff
@@ -0,0 +1,400 @@
+# The Book of Self-Repair (Heartbeat First Impressions)
+
+##  Overall Goals of Self-Repair
+
+1.  Host self-repair recipes, outside of firefox core.  Run them with Firefox permissions.  Allow fast development off the trains.
+
+2.  Collect randomly sampled User Sentiment scores.  Implemented as "Heartbeat First Impressions" recipe.
+
+
+## As a developer, I want to successfully hack on Self-Repair
+
+### Setup (do once)
+
+1.  Zen Of Cooking
+
+	This is the short version. `Cmd+F` will find bigger explanations of nearly everything later on in this document.  
+
+   Mastering this may take several attempts.  Skip around.  Be gentle on yourself!  Enjoy the process!
+   
+1. Pots and Pans
+
+	```
+	# required: git, node
+	brew install node   # includes npm
+	brew install git    # git
+	
+	# optional:  hub, git
+	brew install hub    # github's replacement git client
+	brew install ghi    # github issues client
+	```
+
+1. Groceries
+
+	```
+	# usual git project clone and fork
+	
+	git clone mozilla/self-repair-server
+	cd self-repair-server
+	hub fork
+	```
+
+1. Prep the veggies
+
+	```
+	# usual node project package install
+	npm install   
+	
+	# installs dependecies at `node_modules`.
+	```
+
+1. Preheat the oven.
+
+	**Firefox Settings** - allow local UITour debug in `about:config`
+	
+	- `browser.uitour.requireSecure;false`
+	- `browser.uitour.testingOrigins;http://localhost`
+
+1.  You are ready to cook.  Awesome job.
+
+### Explore
+
+1.  Open the page (this is safe)
+
+    `npm run open-prod`
```

Comment:
Oh, I see! I'm looking at the branch in this pull request which does not have that script:

https://github.com/gregglind/self-repair-server/blob/bdbce9eef9bf5237db37e1b1057e024511beb33c/package.json#L43

I thought you were putting all the changes in this branch--my bad. I'll look at master.